src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            if (line.startsWith("type")) {
+                continue;
+            }
+            String[] parts = line.split(",");
```

Comment:
It's a good practice to define the delimiter as a constant to avoid magic strings and make it clear what it represents.